### ISSUE TITLE
This can get HOTA 69.9?

### ISSUE THREAD

**Issue by paperkaiser** [megvii-research/MOTRv2#2]
because i look for the training steps you provided and the use of yolox ，but can't get the HOTA of 69.9 ,So I want to ask if the code you provided can get 69.9, Thanks!!!

**Comment by zyayoung** [comment]
This repo is theoretically the same as the one that gets 69.9 test HOTA, except some cleanups.
We do observe a relatively high instability (~2pt). Our latest run of this repo yields a 68.33 test HOTA on DanceTrack.

**Comment by paperkaiser** [comment]
> This repo is theoretically the same as the one that gets 69.9 test HOTA, except some cleanups. We do observe a relatively high instability (~2pt). Our latest run of this repo yields a 68.33 test HOTA on DanceTrack.

thanks for your reply！

**Comment by zyayoung** [comment]
> the use of yolox ，but can't get the HOTA of 69.9

If you add YOLOX proposal to the original MOTR repo, you may need to modify track query filtering in QIM and tune inference hyperparameters (score threshold and miss tolerance).
You may refer to the "Implementation Details" section for details.

**Comment by wenyuqing** [comment]
> This repo is theoretically the same as the one that gets 69.9 test HOTA, except some cleanups. We do observe a relatively high instability (~2pt). Our latest run of this repo yields a 68.33 test HOTA on DanceTrack.

Hi, I'm curious about what factors cause the instability as the random seed is fixed.
